packages/bundler/src/commands/inline.js

```diff
@@ -123,20 +126,21 @@ async function inlineSassDependencies(
 function findSassModule(packageName, cwd) {
   let currentDirectory = cwd;
 
-  while (currentDirectory !== '/') {
-    const nodeModulesFolder = path.join(currentDirectory, 'node_modules');
+  if (
```

Comment:
Seems like we lost the recursive nature of this block, any particular reason?

Separately, what are the best practices for handling per-drive directory naming conventions? 🤔

Reply:
On loosing the recursion, that's my bad, I've had another look into the logic & it seems I misunderstood it.

On the windows per-drive directory naming, I found that they start with any letter followed by the ``://`` or ``:\`` in nodes case, hence the check for that with a regular exp.